app/services/locations/finder.rb

```diff
@@ -2,25 +2,29 @@
 
 module Locations
   class Finder
-    attr_accessor :filter_params, :order_by, :order_direction
+    attr_accessor :filter_params, :order_by, :order_direction, :active_record_relationships
 
-    def initialize(filter_params, sort_params = {})
+    def initialize(filter_params:, sort_params: {}, active_record_relationships: [])
```

Comment:
Swapped to named parameters which makes the PR a little noisy but is worth the improved clarity I think